### DISCUSSION TITLE
i have issues after compiled Edge

### DISCUSSION THREAD

**Discussion by drdedou** [seleniumbase/SeleniumBase#2587] [Q&A] [answered]
i use pyinstaller to complied

first issue : when i run with edge i have this issue ![received_417615673967958.jpeg](https://github.com/seleniumbase/SeleniumBase/assets/47856801/738afdfd-be2b-4a68-8b07-2ff4f27cf73d)

second issue : run as mobile don't work after compiled 

my configuration : 
with SB(uc=True,undetected=True,browser=typeOfBrowser,start_page=GetUrl(),maximize=True,agent=UserAgent,locale_code=LocaleCode ,mobile=True) as sb:

The program works fine in debug mode

**Comment by mdmintz** [comment, marked as answer]
UC Mode doesn't support Edge
